ui/src/ensembler/components/modal/DeleteEnsemblerModal.js

```diff
@@ -54,14 +54,16 @@ export const DeleteEnsemblerModal = ({
   const modalClosed = () => {
     setCanDeleteEnsembler(true)
     setEnsemblerUsedByCurrentRouterVersion(false)
+    setEnsemblerUsedByActiveRouterVersion(false)
+    setEnsemblerUsedByActiveEnsemblingJob(false)
```

Comment:
These two setters are introduced to ensure the states are being reset to their default values.